extensions/amp-story/1.0/page-advancement.js

```diff
@@ -476,8 +476,8 @@ export class ManualAdvancement extends AdvancementConfig {
       dev().assertElement(event.target),
       (el) => {
         tagName = el.tagName.toLowerCase();
-
-        if (tagName === 'amp-story-page-attachment') {
+        
+        if (el.classList.contains('i-amphtml-page-attachment-host')) {
```

Comment:
This is about page attachments, not page attachment opening UI. Any tap within the content of a page attachment drawer will navigate to the story page now.

Please keep this as it was, and simply add a `role=button` to the shadow host of the opening UI, this will prevent navigation.

Reply:
This was resolved when we synced #33079